compiler_opt/rl/inlining/gin_configs/common.gin

```diff
@@ -7,3 +7,9 @@ clang_path=None
 runners.InliningRunner.llvm_size_path=%llvm_size_path
 runners.InliningRunner.clang_path=%clang_path
 runners.InliningRunner.launcher_path=%launcher_path
+
+add_flags=()
```

Comment:
why define variables for it?

Reply:
so as to abstract away the gin modules and function hierarchy for users when providing flags via command line. this keeps usage simple:

`--gin_files=third_party/ml_compiler_opt/compiler_opt/rl/regalloc/gin_configs/common.gin --gin_bindings='add_flags=("flag1","flag2")'`

Reply:
--gin_bindings='problem_config.flags_to_add.add_flags=("flag1", "flag2")' would also work? in this sense it provides more information on what the flag is about.

Reply:
Done.